File: backend/player_data.py
players = ['yellow', 'red', 'blue', 'green']

# there are 21 pieces, each has a set index; True means the player has the piece
player_pieces = {
    'yellow': [True] * 21,
    'red': [True] * 21,
    'blue': [True] * 21,
    'green': [True] * 21,
}

# which pieces can actually be played
playable_pieces = {
    'yellow': [True] * 21,
    'red': [True] * 21,
    'blue': [True] * 21,
    'green': [True] * 21,
}

# who is still in the game
can_play = {
    'yellow': True,
    'red': True,
    'blue': True,
    'green': True,
}

File: frontend/src/gameLogic/player_data.js
export let players = ['yellow', 'red', 'blue', 'green'];

// there are 21 pieces, each has a set index; true means the player has the piece
export let player_pieces = {
    yellow: Array(21).fill(true),
    red: Array(21).fill(true),
    blue: Array(21).fill(true),
    green: Array(21).fill(true),
}

// which pieces can actually be played
export let playable_pieces = {
    yellow: Array(21).fill(true),
    red: Array(21).fill(true),
    blue: Array(21).fill(true),
    green: Array(21).fill(true),
}

// who is still in the game
export let can_play = {
    yellow: true,
    red: true,
    blue: true,
    green: true,
}